stable/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 7.1.6
+version: 7.1.7
```

Comment:
I think we should stick to semantic versioning, as this includes new features, we should bump to `7.2.0`. 

Just add a quick blurb about what was added in `UPGRADE.md` and link to it from `README.md`.